packages/fxa-settings/src/pages/Signup/index.test.tsx

```diff
@@ -107,6 +108,46 @@ describe('Signup page', () => {
     expect(firefoxPrivacyLink).toHaveAttribute('href', '/legal/privacy');
   });
 
+  describe('hardNavigateToContentServer', () => {
+    let hardNavigateToContentServerSpy: jest.SpyInstance;
+    // Not needed once this page doesn't use `hardNavigateToContentServer`
+    const originalWindow = window.location;
```

Comment:
Do we need the window setup and `beforeAll`, `afterAll` blocks? I ran the test without these (only keeping `hardNavigateToContentServerSpy`) and all passed.

Reply:
Ohh interesting, good catch. 

Any other time we've tested a file using `hardNavigateToContentServer`, we've mocked it this way. If I remove the setup/cleanup from another test file with this, the test fails. However, I wanted to check what the function was called with in this test, so I used a spy instance. Looks like if we use the spy instance, we don't need the window mocks. I'll remove!